exchanges/huobi/huobi_wrapper.go

```diff
@@ -385,31 +531,73 @@ func (h *HUOBI) FetchOrderbook(p currency.Pair, assetType asset.Item) (*orderboo
 
 // UpdateOrderbook updates and returns the orderbook for a currency pair
 func (h *HUOBI) UpdateOrderbook(p currency.Pair, assetType asset.Item) (*orderbook.Base, error) {
```

Comment:
[ERROR]|ORDERBOOK| 29/12/2020 16:09:38 |Failed to get REST orderbook. Error: orderbook bids and asks are empty.
I think this is for a futures contract: ETC-210625.

Coinmarginedfutures are working correctly.